config-server/.mvn/wrapper/maven-wrapper.properties

```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.5.0/apache-maven-3.5.0-bin.zip
```

Comment:
Should we use the latest?

Reply:
We can do a bump of all projects at the same time